.goreleaser.yaml

```diff
@@ -0,0 +1,64 @@
+builds:
```

Comment:
Note for @richardcase since you asked about it before - the `changelog.disabled` flag actually disabled posting a changelog/release notes entirely; even customized ones passed via `--release-notes`. The `--release-notes` flag is what overrides the Goreleaser changelog generation.

See https://goreleaser.com/customization/changelog/

Reply:
Noted and understood, thanks @nrb . I mentioned it more in the context that currently the makefile would generate and attach the release notes. But if we are attaching the custom release notes directly with goreleaser then great :+1: